.run/Backend Okta.run.xml

```diff
@@ -0,0 +1,13 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Backend Okta" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
+    <option name="ACTIVE_PROFILES" value="okta-local,local" />
+    <envs>
+      <env name="OKTA_API_KEY" value="$OKTA_API_KEY" />
```

Comment:
Adding this here to indicate to people that they have to set `OKTA_API_KEY` in their `local env` since its required to run with OKTA. Note, I didn't supply it here for security reasons and didn't want to add a dummy value like `REPLACE_ME` to prevent people from committing the real value by accident when they replace it